project/plugins.sbt

```diff
@@ -1,8 +1,10 @@
-addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "1.16.0")
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.16.0")
 addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.3.2")
-addSbtPlugin("org.scalameta"      % "sbt-scalafmt"             % "2.4.4")
+addSbtPlugin("org.portable-scala" % "sbt-scala-native-crossproject" % "1.3.2")
+addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.4")
 
-addSbtPlugin("com.github.sbt" % "sbt-ci-release"     % "1.5.10")
-addSbtPlugin("ch.epfl.scala"  % "sbt-version-policy" % "2.1.0")
+addSbtPlugin("com.github.sbt" % "sbt-ci-release" % "1.9.3")
```

Comment:
i did this upgrade because cousier couldn't resolve a transitive dependency - no idea if this breaks anything